tests/schemes/test_thrift.py

```diff
@@ -341,7 +344,7 @@ def testStruct(request, response, proxy):
         # TODO server getting headers in non-friendly format,
         # create a top-level request that has friendly headers :)
         # assert request.headers == {'req': 'headers'}
-        assert request.headers == [['req', 'header']]
+        assert request.headers == {'req': 'header'}
```

Comment:
What did you change so that headers started coming in this format? You'll want to record it as a breaking change in the changelog and detail the upgrade path in upgrade.rst.